### ISSUE TITLE
Readme: example of diff output

### ISSUE THREAD

**Issue by TehShrike** [concordancejs/concordance#27]
The docs say "Concordance strives to format every aspect of a value that is used for comparisons. Formatting is optimized for human legibility", but never give any examples of diff output - I'm a lazy person  just browsing around libraries on Github, so I'm not motivated enough to install the library and poke at it in the REPL, but I am curious as to what the library does.

Examples of human-readable object/value diffing in the readme would be useful.

**Comment by novemberborn** [comment]
Agreed. I just haven't had the time yet to add examples. We'll be shipping this in [AVA](https://github.com/avajs/ava/) soon so that'll help.

**Comment by sindresorhus** [comment]
Until the readme is updated, you can see some examples here: https://twitter.com/ava__js/status/880125395680546816

**Comment by jasikpark** [comment]
It would also be great to have documentation for how to use this 😅
